### ISSUE TITLE
you will be change on 11900K?

### ISSUE THREAD

**Issue by alexkada** [SchmockLord/Hackintosh-Intel-i9-10900k-Gigabyte-Z490-Vision-D#152]

**Comment by SchmockLord** [comment]
Already did. https://github.com/SchmockLord/Gigabyte-Z590i-Vision-D-11900k

**Comment by gtrooper** [comment]
> Already did. https://github.com/SchmockLord/Gigabyte-Z590i-Vision-D-11900k

Hi SchmockLord ! I own the Z490VisionD and just want to ask you if you tried to use 11900k with Z490VisionD ! I need to update 10900K to 11900K to use PCIe 4.0 on one of the three NVME SSD's Z490VisionD has. I just did the BIOS update that will allow me to change the processor when it comes. Do you know what changes I need to do in this config to achieve this ? I assume that the only thing I need to change is the CPU-ID  to the one you used on Z590 Build ? Thank you for this amazing EFI !

**Comment by SchmockLord** [comment]
No. I don't own a Z490 Vision D anymore.

And you won't be able to use PCI4.0 with the 11900k on a Z490 Board. Intel wants you to buy a Z590 for PCIE4.0 as well.

**Comment by gtrooper** [comment]
In the official site they state that the 1st m2 PCIe Slot from Up To Down supports an m2 NVME. Also the 1st PCIe Slot is a PCIe 4.0 x16 (Supposedly for the Graphics Card). Was this just marketing so that you could invest  early-on ? I hope not ! You can check the image I am attaching and tell me what you think ! I have no problem selling it and getting the Z590 but I the only PCIe use cause I need is ONE m2 PCIe 4.0 slot I dont think it is necessary ! Your opinion on this ? Am I missing something ? Thanks !  https://i.imgur.com/v7f392I.png

**Comment by SchmockLord** [comment]
Yeah, I think the board manufacturers where marketing PCI4.0 and the boards technically can it, but Intel said "Nah, you need a 11th Gen AND a Z590 board for that."

**Comment by gtrooper** [comment]
Oh, I see ! Thanks for the answer !

**Comment by SchmockLord** [comment]
@gtrooper I have to correct what I said earlier. I just found out, that Gigabytes Z490 are fully supporting PCI4.0 with 11th Gen Rocket Lake.

See here: https://www.gigabyte.com/de/Press/News/1870

**Comment by gtrooper** [comment]
oh, A Huge Thanks for confirming that to me ! One less component to update I guess ! In the next few weeks I'll get the 11900K and confirm this. Thank you, SchmockLord ! Just out of curiosity... Why did you choose the Z590 Vision i D and not the Bigger Z590 ATX as you did here with Z490 ?

**Comment by SchmockLord** [comment]
You're welcome! Because I have switched to miniITX about half a year ago.
